### PR TITLE
Fix/ong permissions and exceptions

### PR DESCRIPTION
**Pull request by pronguto**

### CODE REVIEW

File: ongs/views.py [hunk 1]
```diff
@@ -23,8 +23,7 @@ def post(self, request: Request):
             raise MissingKeyException("causes", "This field is required.")
 
         if not isinstance(causes, list):
-            # TODO create an exception to handle this properly
-            raise Exception("Not a valid list of strings.")
+            raise KeyTypeError(key="causes", message="Expect a list of itens")
```
Comment (marcelosj3) [inline ongs/views.py:26]: typo, should be `items`

File: ongs/views.py [hunk 2]
```diff
@@ -46,7 +45,10 @@ def patch(self, request: Request, ong_id):
         try:
             ong = get_object_or_404(Ong, pk=ong_id)
             self.check_object_permissions(request, ong)
-            if request.data.get("causes", None):
+            causes = request.data.get("causes", False)
+            if causes:
+                if not isinstance(causes, list):
+                    raise KeyTypeError(key="causes", message="Expect a list of itens")
```
Comment (marcelosj3) [inline ongs/views.py:51]: Same as above

**Review by marcelosj3 — APPROVED** [review]
Since the typo is the only flaw, i'll approve

**Review by marcelosj3 — APPROVED** [review]

File: ongs/views.py (hunk 1 above)
Comment (pronguto) [reply ongs/views.py]: Ok

File: ongs/views.py (hunk 2 above)
Comment (pronguto) [reply ongs/views.py]: Ok

**Review by marcelosj3 — APPROVED** [review]

**Review by gabrielsuch — APPROVED** [review]

**Pull request merged**